docker-entrypoint.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+export CONFIG_DIR="${CONFIG_DIR:-/opt/config}"
+export CONFIG=$CONFIG_DIR/config.json
+export FORCE_BUILD_BEFORE_START="${FORCE_BUILD_BEFORE_START:-false}"
+
+# Supported verbs: install (web install), setup (interactive CLI session). Default: web install
+# TODO: constraint it using a hash set (or hash table)
+export NODEBB_INIT_VERB="${NODEBB_INIT_VERB:-install}"
+# Setup variable for backward compatibility, default: <empty>
+export SETUP="${SETUP:-}"
+
+mkdir -p $CONFIG_DIR
+chmod -fR 760 $CONFIG_DIR
```

Comment:
I'd redirect the errors from this to `/dev/null` - volumes on at least some platforms mount owned by root, making this fail despite the folder still being writable.

Though I'd also add a comment explaining this to help debugging. E.g.
```bash
# If the folder is mounted as a volume this can fail, the check below is to ensure there is still write access
chmod -fR 760 $CONFIG_DIR 2> /dev/null
```